packages/e2e-tests/fixtures/blocks/README.md

```diff
@@ -34,16 +39,45 @@ this way.  New contributions in the form of additional test cases are always
 welcome - this is a great way for us to identify bugs and prevent them from
 recurring in the future.
 
-When adding a new test, it's only necessary to create file (1) above, then
-there is a command you can run to generate (2) through (4):
+## Creating Fixtures
+
+When adding a new fixtures, only the first file above (1, e.g. `core__image.html`) needs
+to be created manually, the other files are generated from this first file. 
+
+To create the first file:
+1. Create a file with the correct name in this folder.
+2. Add the block to an new post in the editor.
+3. Toggle the block attributes to desired settings for the test.
+4. Switch to the code editor view and copy the block markup.
+5. Paste the markup into the file you created at step 1.
+
+Next, to generate files (2) through (4) run the following command from the root of the
+project:
 
 ```sh
 GENERATE_MISSING_FIXTURES=y npm run test-unit test/integration/full-content/full-content.test.js
 ```
 
-However, when using this command, please be sure to manually verify that the
+When using this command, please be sure to manually verify that the
 contents of the `.json` and `.serialized.html` files are as expected.
 
+In particular, check that the `isValid` property is `true`, and that
+the attributes are serialized correctly.
+
+## Updating Fixtures
+
+The process for updating fixtures for existing tests is similar to that for creating them:
+
+First delete files (2) through (4), then run the command to regenerate the files:
```

Comment:
There are scripts defined that make it easier:

https://github.com/WordPress/gutenberg/blob/c31475afc9fc149262da67c7464d5f4a527eb93a/package.json#L208-L210

Maybe we should scope them to this folder and simplify those documents. All you need is really:

```bash
$ npm run fixtures:regenerate
```

Reply:
It also reminds me that we moved fixtures to e2e tests to be able to reuse them. The question is if it is still the case. Maybe we should move them back to live closer to the tests that use them. @jorgefilipecosta do you know the latest status?

Reply:
@gziolo I always felt a bit old school using that `GENERATE_MISSING_FIXTURES` variable 😄 

The suggestion is a lot nicer. In #22661 I've updated the docs to use `npm run fixtures:regenerate` with the path to the file so that they run a bit faster.

Reply:
Currently, we are not reusing the fixtures in e2e tests, so I guess we may move them. But allowing the possibility to reuse in the future is not necessarily bad.